### ISSUE TITLE
Chart outdated

### ISSUE THREAD

**Issue by 1Map** [victorgarciaesgi/vue-chart-3#139]
According to your package.json
"chart.js": "=> ^3.1.0",

This version is a 2 year old version (4.2.0 is the latest)

**Comment by BayBreezy** [comment]
Any updates on this? It stops a normal npm install

**Comment by DominikGanic** [comment]
Yeah, sadly, think we have to move on from this package now. Can't use it with "chartjs-plugin-crosshair" as the v2 requires chart.js 4.0.1 or higher - and this is not supported here!

**Comment by inkbeard** [comment]
Based on https://github.com/victorgarciaesgi/vue-chart-3?tab=readme-ov-file#im-joining-vue-chartjs-as-a-core-maintainer-so-this-package-will-be-kept-just-for-those-who-already-use-it-im-encouraging-new-users-to-use-vue-chartjs-instead, this is not being maintained any longer and you should use https://github.com/apertureless/vue-chartjs instead, which is actively being updated as of 12/14/23.
